runtime.ts

```diff
@@ -1149,7 +1206,18 @@ module J2ME {
   }
 
   function findCompiledMethod(klass: Klass, methodInfo: MethodInfo): Function {
-    return jsGlobal[methodInfo.mangledClassAndMethodName];
+    var name = methodInfo.mangledClassAndMethodName;
+    var method = jsGlobal[name];
+    return method;
+    if (!method) {
+      var compiledMethod = baselineCompileMethod(methodInfo, CompilationTarget.Runtime);
```

Comment:
> This looks like a JIT compilation step that has been disabled. If I enable it, tests hang. Is it a dead end? If so, we should remove it. Otherwise, we should make it work.

Okay, I'll remove it form my branch.